scripts/shell/init_node_template.sh

```
@@ -0,0 +1,140 @@
+
+
+$BIN add-genesis-account $ARGS $VALIDATOR_KEY1 $ALLOCATION
+
+
+# Update account registry
+FILE="/data/account.json"
+sudo jq --arg key1 "$KEY1" --arg key2 "$KEY2" \
+    --arg validator_key "$VALIDATOR_KEY1" \
+    --arg orchestrator_key "$ORCHESTRATOR_KEY1" \
+    '. + {($key1): $validator_key, ($key2): $orchestrator_key}' \
+    "$FILE" > tmp.json && sudo mv tmp.json "$FILE"
```

Shellcheck (0.11.0)

[warning] sudo doesn't affect redirects. Use ..| sudo tee file

(SC2024)